app/models/ambassador_task.rb

```diff
@@ -0,0 +1,23 @@
+class AmbassadorTask < ActiveRecord::Base
+  has_many :ambassador_task_assignments
+  has_many :users, through: :ambassador_task_assignments
+
+  validates :description, uniqueness: true
+
+  def description_html
+    Kramdown::Document.new(description).to_html
+  end
+
+  def assign_to(user)
+    return unless user.ambassador?
```

Comment:
This will hide errors. e.g. if I try `assign_to(user)` and that user is not an ambassador, the validation you added will stop the creation, and you'll be able to see why - if you do this, it will instead return nil instead of a task, and you won't see any errors.

Reply:
Something additional this method could handle is making sure we don't assign the same task to the same user multiple times